app/containers/Vaults/saga.js

```diff
@@ -61,14 +62,65 @@ const injectEthVaults = (vaults) => {
   return vaults;
 };
 
+const mapNewApiToOldApi = (oldVaults, newVaults) => {
+  const newVaultsMap = keyBy(newVaults, 'address');
+  const result = oldVaults.map((vault) => {
+    const newApy = get(newVaultsMap[vault.address], 'apy');
+    console.log({ oldAPI: vault });
+    console.log({ newAPI: newVaultsMap[vault.address] });
+    if (!newApy) {
+      console.log('------------');
+      return vault;
+    }
+
+    const mergedVault = {
+      ...vault,
+      apy: {
+        ...vault.apy,
+        recommended: newApy.net_apy,
+        data: {
+          ...vault.apy.data,
+          grossApy: newApy.gross_apr,
+          netApy: newApy.net_apy,
+          ...(newApy.composite && {
+            totalApy: newApy.composite.pool_apy
```

Comment:
we should really validate this with all the type of vaults

Reply:
changed to use gross_apr @dudesahn